cms/core/blocks/markup.py

```diff
@@ -114,6 +114,11 @@ class ONSTableBlock(TinyTableBlock):
     source = blocks.CharBlock(label="Source", required=False)
     footnotes = blocks.RichTextBlock(label="Footnotes", features=settings.RICH_TEXT_BASIC, required=False)
 
+    def __init__(self, *, local_blocks=None, search_index=True, **kwargs):
```

Comment:
To resolve the linter failure
```suggestion
    from wagtail.blocks.base import Block
    def __init__(self, *, local_blocks: list[Block] | None = None, search_index: bool = True, **kwargs: Any) -> None:
```

Reply:
argh, good shout! 06077ec